src/container.ts

```diff
@@ -90,7 +90,7 @@ container.register({
 const SAML_CALLBACK_URL =
   process.env.SAML_CALLBACK_URL ||
   "http://italia-backend/assertionConsumerService";
-const SAML_ISSUER = process.env.SAML_ISSUER || "http://italia-backend";
+const SAML_ISSUER = process.env.SAML_ISSUER || "http://italiabackend.it";
```

Comment:
why this one ?

Reply:
because in the new testenv2 the issuer has to be a valid URL

Reply:
is there any reason on why this differs from the default saml issuer (https://spid.agid.gov.it/cd) ?
I mean, cant we just put https://spid.agid.gov.it/cd into the sp_metatadata.xml of the test spid IDP ?